README.md

```diff
@@ -32,11 +32,11 @@ Add the SDK package to your `package.json` file.
 
 #### Using Yarn
 
-`yarn add @pngme/react-native-sms-pngme-android@2.0.4`
+`yarn add @pngme/react-native-sms-pngme-android`
```

Comment:
We should keep a specific version pinned, to avoid version conflicts between native dependency versions specified in `android/app/build.gradle` and the Pngme SDK.